app/views/WhenReceiveShortageExcessView.scala.html

```diff
@@ -0,0 +1,65 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import models.requests.DataRequest
+
+@this(
+        layout: templates.Layout,
+        formHelper: FormWithCSRF,
+        govukErrorSummary: GovukErrorSummary,
+        govukDateInput: GovukDateInput,
+        govukButton: GovukButton,
+        h1: components.h1,
+        h2: components.h2,
+        p: components.p
+)
+
+@(form: Form[_], action: Call)(implicit request: DataRequest[_], messages: Messages)
+
+@layout(pageTitle = title(form, messages("whenReceiveShortageExcess.title"))) {
+
+    @formHelper(action, 'autoComplete -> "off") {
+
+        @if(form.errors.nonEmpty) {
+            @govukErrorSummary(ErrorSummaryViewModel(form, errorLinkOverrides = Map("value" -> "value.day")))
+        }
+
+        @h1("whenReceiveShortageExcess.heading", Some(messages("arc.subHeading", request.arc)), "govuk-heading-l")
+
+        @p() {
+            @messages("whenReceiveShortageExcess.p1")
+        }
+
+        @p() {
+            @messages("whenReceiveShortageExcess.p2")
+        }
+
+        @govukDateInput(
+            DateViewModel(
+                field  = form("value"),
+                legend = LegendViewModel(HtmlContent(legend)).withCssClass("govuk-fieldset__legend govuk-fieldset__legend--l")
+            ).withHint(HintViewModel(messages("whenReceiveShortageExcess.hint")))
+        )
+
+        @govukButton(
+            ButtonViewModel(messages("site.saveAndContinue"))
```

Comment:
Copydeck only has "Continue" - need to double check whether the prototype is wrong or if the copydeck needs updating.

Reply:
Copy deck is wrong, me and Ste checked with Tom and Clare